### ISSUE TITLE
Suggestion: A way to copy the jb2a DB entry referenced by an AA Autorec (AR) entry...

### ISSUE THREAD

**Issue by dstein766** [otigon/automated-jb2a-animations#405]
...similar to what is available on the JB2A Asset Library site.  The use case: I use Sequencer.Preloader.preloadForClients to pre-cache a lot of stuff to my players' side.  This involves passing an array of assets to be cached, many of which are jb2a DB paths.  So, if (e.g.) I have a PC that uses spell X a lot, I want to preload whatever asset is defined in AR for that spell.  Currently, I have to look at the setting in AR, then go to JB2A's Asset Library and try to figure out which asset is matched so I can copy it from JB2A's site and paste into my script.  It is sometimes a challenge to match up assets, and it is always an extra step.  It would be awesome if I could simply click a button in AR and it would copy the underlying JB2A DB path to clipboard (or expose it as selectable text, etc.)  [BTW: I'm weeks behind in this channel, so if already discussed my apologies]

**Comment by otigon** [comment]
I'll add this in while I'm reworking the menus

**Comment by otigon** [comment]
https://user-images.githubusercontent.com/73318661/167761316-69f3b233-24eb-4a16-8ec7-8f72e81273ff.mp4

It will essentially be this. Though there's really now good way to specify a determinative color if they have chosen Random. That DB path would cause quite a few to pre-load

**Comment by otigon** [comment]
Also note that the DB paths in A-A cannot be accessed in the Sequencer DB viewer, as I keep it hidden due to being more apt to changes in structure

**Comment by leonn3k** [comment]
Would it not be simpler from a user perspective to preloadForClientsb animations/audio right before the sequence is ran? Or at least an option for it to happen would be useful. I've been making a macro to handle an edgecase and it adds a lot of "stability" when using multiple animations or syncing animations with audio while having a minimal delay for clients. Perhaps that should have an issue of its own, though.

**Comment by otigon** [comment]
Yeah that would be a separate issue. I've thought about it in the past, but it bogs things down for things like range attacks because that's 5 different files that would preload just for an arrow, and then multiplied if they have a random color chosen

**Comment by leonn3k** [comment]
Perhaps you could handle the random choice in your own code rather than through the sequencer database .random? Then you have a variable you can feed into preload. You could always have a global and a per anim choice for it, I guess. Would be nice if there was a way to have files preload from inside the sequence since it'd already know what choice it's making. I'll try not to spam this post too much more but I do see why it's more complicated than I first thought.

**Comment by otigon** [comment]
It would take a hefty amount of work on my end, so it wouldn't be implemented (if at all) for some time

**Comment by otigon** [comment]
I left out the Database path copy on the Version 4 release, but I'll take a look at adding it back in. It was buggy at best and was lower on the totem pole for features I wanted to get sorted out
